src/less/radio/ds6/radio-base.less

```diff
@@ -4,88 +4,35 @@
     vertical-align: text-bottom;
 }
 
-.radio__control[type="radio"],
-.radio__custom-control[type="radio"] {
+.radio__control[type="radio"] {
     font-size: 100%;
     height: 1.125em;
     opacity: 0;
     position: absolute;
     width: 1.125em;
     z-index: 1;
-}
-
-.radio__custom-control[type="radio"]:not([disabled]) {
-    cursor: pointer;
-}
-
-.radio__control[type="radio"] {
-    &:checked + .radio__icon use {
-        fill: @ds6-color-p003-blue;
-    }
-
-    &:checked + .radio__icon:not([hidden]) {
-        .background-icon-base;
-        .icon-radio-checked();
-    }
-
-    + .radio__icon--inherit {
-        color: inherit;
-    }
-
-    &:checked + .radio__icon--inherit use {
-        fill: inherit;
-    }
-
-    &:focus + .radio__icon {
-        outline: @outline-dotted;
-    }
-
-    &[disabled] + .radio__icon {
-        opacity: 0.5;
-    }
 
     &:not(:checked) {
-        & + .radio__icon .radio__checked {
+        & + .radio__icon svg.radio__checked {
             display: none;
         }
 
-        & + .radio__icon .radio__unchecked {
+        & + .radio__icon svg.radio__unchecked {
             display: inline-block;
         }
     }
 
     &:checked {
-        & + .radio__icon .radio__checked {
-            display: inline-block;
+        & + .radio__icon:not([hidden]) {
+            .background-icon-base;
```

Comment:
Thinking out loud here: can we add `.background-icon-base` to all the mixins within `mixins.less`?

Reply:
I don't think so, because we only want those styles to be applied to the base class (i.e. `.icon`)

Reply:
Ah, these mixins are used elsewhere, without those rules. Gotcha.